docs/resources/logdna_view.md

```diff
@@ -73,50 +73,50 @@ resource "logdna_view" "my_view" {
-- `hosts`: (Optional) Array of names of hosts to filter the view by
-- `apps`: (Optional) Array of names of apps to filter the view by
-- `levels`: (Optional) Array of names of levels to filter the view by
-- `tags`: (Optional) Array of names of tags to filter the view by
+- `name`: (Required) The name this View will be given
+- `query`: (Optional) The search query scope for the View
+- `categories`: (Optional) An array of existing category names this View should be nested under. Not case sensitive. Note: if the category does not exist- the View will by default be created in uncategorized
+- `hosts`: (Optional) Array of names of hosts to filter the View by
+- `apps`: (Optional) Array of names of apps to filter the View by
+- `levels`: (Optional) Array of names of levels to filter the View by
+- `tags`: (Optional) Array of names of tags to filter the View by
 
 ### email_channel
 
 `email_channel` supports the following arguments:
 
-- `emails`: (Required) An array of emails to notify in the alert
-- `triggerlimit`: (Required) Number of lines before the alert is triggered. (ex. Setting a value of `10` for an `absence` alert would alert you if `10` lines were not seen in the `triggerinterval`)
+- `emails`: (Required) An array of emails to notify in the Alert
+- `triggerlimit`: (Required) Number of lines before the Alert is triggered. (ex. Setting a value of `10` for an `absence` Alert would alert you if `10` lines were not seen in the `triggerinterval`)
 - `timezone`: (Optional) Which time zone the log timestamps will be formatted in.
-- `immediate`: (Optional) _Default: false_ Whether the alert will trigger immediately after the trigger limit is reached.
-- `operator`: (Optional) _Default: Presence_ Whether the alert will trigger on the presence or absence of logs.
-- `terminal`: (Optional) If the alert will trigger after the `triggerinterval` if the alert condition is met (ex. Send an alert after 30s).
-- `triggerinterval`: (Optional) Interval which the alert will be looking for presence or absence of log lines. For `presence` alerts, valid values are: `30s`, `1m`, `5m`, `15m`, `30m`, `1h`, `6h`, `12h`, and `24h` (default `30s`). For `absence` alerts, valid values are:  `15m`, `30m`, `1h`, `6h`, `12h`, and `24h`. (default `15m`)
+- `immediate`: (Optional) _Default: false_ Whether the Alert will trigger immediately after the trigger limit is reached.
+- `operator`: (Optional) _Default: Presence_ Whether the Alert will trigger on the presence or absence of logs.
+- `terminal`: (Optional) If the Alert will trigger after the `triggerinterval` if the Alert condition is met (ex. Send an Alert after 30s).
+- `triggerinterval`: (Optional) Interval which the Alert will be looking for presence or absence of log lines. For `presence` Alerts, valid values are: `30s`, `1m`, `5m`, `15m`, `30m`, `1h`, `6h`, `12h`, and `24h` (default `30s`). For `absence` Alerts, valid values are:  `15m`, `30m`, `1h`, `6h`, `12h`, and `24h`. (default `15m`)
```

Comment:
Sorry I made a mistake in the original docs where `30s` is supposed to be actually `30`

```suggestion
- `triggerinterval`: (Optional) Interval which the Alert will be looking for presence or absence of log lines. For `presence` Alerts, valid values are: `30`, `1m`, `5m`, `15m`, `30m`, `1h`, `6h`, `12h`, and `24h` (default `30`). For `absence` Alerts, valid values are:  `15m`, `30m`, `1h`, `6h`, `12h`, and `24h`. (default `15m`)
```

Reply:
All values are entered in as strings. And all the values are with a unit (other than the seconds value for which it is just 30). 

Could you offer an example of what the documentation for a given field should look @nimbinatus?- am not sure what the suggestion is for where to place the default value.

Reply:
So how exactly to do it is a question for @tanberry as she probably has a standard she'd like to follow. I've used tables in the past, like this:

Key | Type | Value Default | Description
--- | --- | --- | ---
`triggerinterval` | string | `"30"` (`presence` Alerts)<br/>`"15m"` (`absence` Alerts) | Interval which the Alert will be looking for presence or absence of log lines.<br/>For `presence` Alerts, valid values are: `30`, `1m`, `5m`, `15m`, `30m`, `1h`, `6h`, `12h`, and `24h`.<br/>For `absence` Alerts, valid values are:  `15m`, `30m`, `1h`, `6h`, `12h`, and `24h`.

For this README for now, I personally would just use italics and lowercase for optional or required, and then bold at the end for defaults, for consistency until a style guide is out (my personal preference, to be fair, but it's as readable as I can think of inline like this):
```markdown
- `triggerinterval`: _(optional)_ Interval which the Alert will be looking for presence or absence of log lines. For `presence` Alerts, valid values are: `30`, `1m`, `5m`, `15m`, `30m`, `1h`, `6h`, `12h`, and `24h`. For `absence` Alerts, valid values are: `15m`, `30m`, `1h`, `6h`, `12h`, and `24h`. **(Defaults: `"30"` for `presence`; `"15m"` for `absence`)**
```
which looks like this:
- `triggerinterval`: _(optional)_ Interval which the Alert will be looking for presence or absence of log lines. For `presence` Alerts, valid values are: `30`, `1m`, `5m`, `15m`, `30m`, `1h`, `6h`, `12h`, and `24h`. For `absence` Alerts, valid values are: `15m`, `30m`, `1h`, `6h`, `12h`, and `24h`. **(Defaults: `"30"` for `presence`; `"15m"` for `absence`)**

Reply:
Am using your latter approach.